### PR TITLE
Revision

### PR DESCRIPTION
**Pull request by SimonGreenhill**
I thought I could easily fix this buildbot error, but it turned out to not be so easy.

### CODE REVIEW

**Comment by SimonGreenhill** [comment]
(I may have made this worse, so feel free to revert)

File: lexibank_savelyevturkic.py [hunk 1]
```diff
@@ -53,19 +54,34 @@ def cmd_makecldf(self, args):
...
+                continue
+            
+            # patch two weird/broken entries:
+            if row['ID'] == '7560':
+                row['TOKENS'] = 'ɕ i v ɘ ʨ'
+            
+            if row['ID'] == '8367':
+                row['ENTRY'] = 'ʒɯl'
+                
+            segs = [segments.get(x, x) for x in row['TOKENS']]
```
Comment (LinguList) [inline lexibank_savelyevturkic.py]: That's it: `row['TOKENS'].split()`, that's what you need to get to a perfect 100% :)

**Comment by LinguList** [comment]
See my comment, and you'll see what happened ;)

**Comment by SimonGreenhill** [comment]
Ahah, fixed :)

**Pull request merged**